src/vs/workbench/browser/parts/editor/editorGroupView.ts

```diff
@@ -132,6 +134,23 @@ export class EditorGroupView extends Themable implements IEditorGroupView {
 	private readonly whenRestoredPromise = new DeferredPromise<void>();
 	readonly whenRestored = this.whenRestoredPromise.p;
 
+	// Editor Group Context
+	private resourceContext!: ResourceContextKey;
+
+	private groupActiveEditorDirtyContext!: IContextKey<boolean>;
+	private groupActiveEditorPinnedContext!: IContextKey<boolean>;
+	private groupActiveEditorFirstContext!: IContextKey<boolean>;
+	private groupActiveEditorLastContext!: IContextKey<boolean>;
+	private groupActiveEditorStickyContext!: IContextKey<boolean>;
+	private groupEditorsCountContext!: IContextKey<number>;
+	private groupActiveEditorAvailableEditorIds!: IContextKey<string>;
+	private groupHasPinnedAndUnpinnedContext!: IContextKey<boolean>;
+
+	private groupActiveEditorCanSplitInGroupContext!: IContextKey<boolean>;
+	private sideBySideEditorContext!: IContextKey<boolean>;
+
+	private groupLockedContext!: IContextKey<boolean>;
+
```

Comment:
Manye of the context keys are now created and updated in one function so I suggest to not have them as `private` properties anymore.